### ISSUE TITLE
Play to / cast devices not visible until jellyfin is rebooted via web interface 

### ISSUE THREAD

**Issue by Silvereye89** [jellyfin/jellyfin#6249]
**Describe the bug**
Play to/ cast devices aren't visible after docker container start up until I manually reboot jellyfin from the web interface. 

**System (please complete the following information):**
 - OS: Synology DSM running on 918+
 - Virtualization: Docker, using the linuxserver jellyfin image 
 - Clients: I tested Chrome on pc and the Android client 
 - Jellyfin Version: 10.7.5
 - Playback: irrelevant for this bug 
 - Installed Plugins: Fanart, kodi sync plug in, thetvdb, intro plug in 
 - Reverse Proxy: synology's built in nginx proxy 
 - Base URL: none
 - Networking: Bridge mode in docker 
 - Storage: local on NAS

**To Reproduce**
For power saving purposes my NAS shuts down during the night and reboots in the morning. It automatically restarts my docker containers. My clients are not visible via cast until I reboot jellyfin from the web interface.
Ps: I even tried a manual reboot of the docker container, I.e. Shutting it down and relaunching it via the synology gui. The same issue persists until I reboot (shutdown button) from jellyfin web interface. 

**Expected behavior**
All connected clients should be visible as play to/ cast targets once the jellyfin docker container started 

**Additional context**
- Does it have something to do with the websockets? 
I am also not able to control, e.g. stop the playback of connected devices until I rebooted jellyfin. 

- I did not have this issue in 10.6.x

Thanks in advance and I am looking forward to suggestions.

**Comment by BaronGreenback** [comment]
Can you post your JF logs please?

**Comment by Silvereye89** [comment]
Hi @BaronGreenback,

Thank you for your reply.
You can find the logs here at [pastebin](https://pastebin.com/VSNsGuXa)

**Comment by BaronGreenback** [comment]
@Silvereye89 Sorry, can't access the log. I'm getting the error, `this is a private paste or is pending moderation. If this paste belongs to you, please login to Pastebin to view it.`

**Comment by Silvereye89** [comment]
@BaronGreenback That's strange. I've set "paste exposure" to "unlisted". Afaik it should be accessible by the link above. But I've tried it myself in an incognito browser now and get the same error. Now I've even tried to set it public with the exact same error message coming up.

Am I doing something wrong? I am using pastebin for the first time.
I'll attach the logfile to this comment instead.
[Jellyfin Log.txt](https://github.com/jellyfin/jellyfin/files/6758514/Jellyfin.Log.txt)

**Comment by BaronGreenback** [comment]
From looking at the log - it looks like DlnaEntryPoint is not listening to interfaces.

Could you edit the `logging.default.json` file and change the default to `debug`?

This will create a mucg bigger log, and should hopefully help to pinpoint the issue.

**Comment by Silvereye89** [comment]
I am not using DLNA at all and have it disabled eversince. And still I was able to cast to my devices before :-)

Could you please specify if I should rename the file itsself or change the entry `"Default": "Information"` to `"Default": "debug"`

**Comment by BaronGreenback** [comment]
should contain "Default": "Debug"

PlayTo is a type of DLNA, so needs to be enabled to pick up DLNA PlayTo clients.
I believe the web interface detects the chrome cast clients - but not having this on my network, i'm not familiar with that.

**Comment by Silvereye89** [comment]
Alright. Attached you'll find the [Jellyfin Debug-Log.txt](https://github.com/jellyfin/jellyfin/files/6758855/Jellyfin.Debug-Log.txt). I rebooted the server via Synology Docker gui right before grabbing the logs in order to clean it up and get the startup entries.

As I said. I turned off everything DLNA related since I installed Jellyfin two years ago. I guess v 10.4.x or something. The clients still did connect fine and were visible via "play to" until the 10.7.x updates. My guess is that it is related to my websockets - even though my local clients connect via LAN-IP instead of going through my reverse proxy.
However, when I reboot the jellyfin server via its webinterface everything works as expected. It's just not working when it is booting up after docker launched it.

Daily manually rebooting via webinterface isn't much of an option though ;-)


Thank you very much for investigating!
